scripts/run_tests.sh

```diff
@@ -40,5 +40,15 @@ display_result $? 2 "Unit tests"
 
 # Run front end tests
 npm config set python python2.7
-yarn test
-display_result $? 1 "Front end tests"
+
+yarn test_unit_no_watch
+display_result $? 1 "Front end unit tests"
+
+SUITE="core"
```

Comment:
It's good to use lowercase variables in bash scripts to avoid possible clashes with env variables